src/de/tnc.md

```diff
@@ -23,12 +23,12 @@ DFX weist ausdrücklich darauf hin, dass ihrerseits keinerlei Anlageberatungsdie
 
```

Comment:
zum Abschnitt ## Keine Beratung für den Kauf von Kryptowährungen

Nur schreiben "Keine Beratung"
und dann im Test darunter Kauf, Verkauf, und co auflisten

DFX empfiehlt Bitcoin oder sonstige Kryptowährungen in keiner Weise.
Nach diesem Satz muss stehen, dass wir eine Ausführungsplattform sind, die von Kunden genutzt werden kann um Transaktionen auszuführen, zu denen sich die Nutzer selbst entschieden haben.